content/1.7/Forms.md

```diff
@@ -0,0 +1,7 @@
+# Specification of forms
+_Context:_
+Rule to be applied after the migration of a from or creation of a new form
```

Comment:
Little typo:
Rule to be applied after the migration of a **from** or creation of a new form => Rule to be applied after the migration of a form or creation of a new form